lib/client/ca_export.go

```diff
@@ -119,11 +119,11 @@ func exportAuth(ctx context.Context, client auth.ClientI, req ExportAuthoritiesR
 		return exportTLSAuthority(ctx, client, req)
 	}
 
-	// If none of the above auth-types was requested, means we are dealing with SSH HostCA or SSH UserCA.
+	// If none of the above auth-types was requested, means we are dealing with SSH HostCA, SSH UserCA, or SSH OpenSSHCA.
```

Comment:
Why, by default OpenSSH CA is exported? It should be only exported when requested explicitly.

Reply:
Changed, but I originally thought since Host and User CAs were exported by default the OpenSSH CA could be too